packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -23,35 +23,170 @@ import fs from "fs";
 
 import { duration } from "../../../tasit-sdk/dist/testHelpers/helpers";
 
-const { ONE, TEN, ONE_HUNDRED, ONE_THOUSAND } = constants;
+const { ONE, TEN, ONE_HUNDRED, ONE_THOUSAND, BILLION, WeiPerEther } = constants;
 
-const createMultipleParcels = async (
-  landContract,
-  parcels,
-  beneficiaryAddress,
-  contractOwnerWallet
-) => {
-  let xArray = [];
-  let yArray = [];
+let network = process.env.NETWORK;
 
-  parcels.forEach(parcel => {
-    xArray.push(parcel.x);
-    yArray.push(parcel.y);
+let EVENTS_TIMEOUT;
+
+const config = require(`../config/${network}.js`);
+
+// https://stats.goerli.net/
+if (network === "goerli") {
+  gasParams = {
+    gasLimit: 8e6,
+    gasPrice: 1e10,
+  };
+  EVENTS_TIMEOUT = 5 * 60 * 1000;
+} else if (network === "ropsten") {
+  EVENTS_TIMEOUT = 5 * 60 * 1000;
+} else if (network === "development") {
+  network = "local";
+  EVENTS_TIMEOUT = 1000;
+}
+const {
+  LANDProxy,
+  EstateRegistry,
+  Marketplace,
+  GnosisSafe: GnosisSafeInfo,
+  MANAToken,
+} = TasitContracts[network];
+const { address: MANA_ADDRESS } = MANAToken;
+const { address: LAND_PROXY_ADDRESS } = LANDProxy;
+const { address: ESTATE_ADDRESS } = EstateRegistry;
+const { address: MARKETPLACE_ADDRESS } = Marketplace;
+const { address: GNOSIS_SAFE_ADDRESS } = GnosisSafeInfo;
+
+ConfigLoader.setConfig(config);
+
+const [minterWallet, sellerWallet] = accounts;
+const { address: sellerAddress } = sellerWallet;
+
+const manaContract = new Mana(MANA_ADDRESS);
+const landContract = new Land(LAND_PROXY_ADDRESS);
+const estateContract = new Estate(ESTATE_ADDRESS);
+const marketplaceContract = new Decentraland(MARKETPLACE_ADDRESS);
+const gnosisSafeContract = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
+
+const provider = ProviderFactory.getProvider();
+
+const cancelSellOrder = async (nftAddress, id) => {
+  const action = marketplaceContract.cancelOrder(
+    nftAddress,
+    `${id}`,
+    gasParams
+  );
+  await action.waitForNonceToUpdate();
+};
+
+const cancelOrdersOfEstatesWithoutImage = async estatesIds => {
+  const getImageDataFromEstateId = async id => {
+    const image = await fetch(
+      `https://api.decentraland.org/v1/estates/${id}/map.png`
+    );
+    const data = (await image.buffer()).toString("base64");
+    return data;
+  };
+
+  // Note: Estate with 5 is one of the estates with blank image
+  const blankImageData = await getImageDataFromEstateId(5);
+
+  marketplaceContract.setWallet(sellerWallet);
+  for (let id of estatesIds) {
```

Comment:
Worth considering doing this in parallel?

Reply:
Not gating for this PR, though.

Most likely we'd want to use a queue data structure where we only process up to n in parallel because we would get throttled if we did too much in parallel

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/295

Reply:
Thanks. Resolving